### ISSUE TITLE
Split §3.1 into 2 parts : explanation on the GSBPM ConceptScheme, and explanations on the class hierarchy

### ISSUE THREAD

**Issue by tfrancart** [linked-statistics/COOS#107]
I suggest:

## § 3.1 The GSBPM activity categories

The GSBPM comprises three levels:

    Level 0, the statistical business process;
    Level 1, the phases of the statistical business process;
    Level 2, the sub-processes within each phase.

A first approach to OWL modelling of statistical activities based on the GSBPM is presented in [[GSBPM-LM](https://linked-statistics.github.io/COOS/coos.html#bib-gsbpm-lm)].

In COOS, a sub-class of skos:Concept named ActivityCategory is created. Each GSBPM phase and sub-process is defined as an instance of ActivityCategory and included (with a skos:inScheme relation) in a skos:ConceptScheme named GSBPM: the scheme of all the GSBPM phase and sub-process categories.

In addition, the statisticalProductionProcess individual, representing the statistical production process as a whole, is also declared as an instance of ActivityCategory and included in the GSBPM concept scheme **as the sole root of the concept scheme**.

The links between the different individuals are represented by the usual SKOS properties: all the sub-processes of a phase have a skos:broader link to the phase, and each phase has a skos:broader link to the statisticalProductionProcess **root concept**.

[example]

(following § moved up in this section):

In addition the ontology defines three individuals of ActivityCategory to model the types of overarching activities provided by GSBPM the “Quality Management”, the “Data Management” and the “Metadata Management”. **They are also linked to the statisticalProductionProcess root concept with a skos:broader property**.

[example]

## § 3.2 Describing actual statistical activities

To materialize a clear distinction between the model itself and the statistical process which is --modeled-- **described**, the ontology defines classes that inherit from prov:Activity. Using PROV allows to describe an activity that really happens at a certain time, to specify who undertakes the activity and what entities it consumes or produces.

(following § need to be rewritten, see #105)

Basically, three classes are defined in this ontology: ProductionActivity, Phase and SubProcess.

ProductionActivity is a sub-class of prov:Activity. Instances of this class are intended to be larger than a GSBPM phase. If they correspond to a statistical production process they are typed using the dcterms:type property with the statisticalProductionProcess individual.

[example]

Phase and SubProcess are both sub-classes of ProductionActivity. A link to the GSBPM taxonomy using the property dcterms:type is made on instances of these classes to formalize to what part of the GSBPM the activity that happened corresponds. Additionally, the idea that each phase is made up of its different sub-processes can be represented by using the Dublin Core dcterms:hasPart and dcterms:isPartOf properties.
